### PR TITLE
read data from file

### PR DESCRIPTION
**Pull request by maks45**

### CODE REVIEW

File: src/main/resources/templates/index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+    <h1>Hello</h1>
+</body>
+</html>
```
Comment (boroda4436) [inline src/main/resources/templates/index.html]: empty line

File: src/test/java/com/maks/durov/amazonreviewapp/service/FileServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        List<String> actual = fileService.readFile(filePath);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void readFileException() {
+        String fakePath = "";
+        assertThrows(DataProcessingException.class, () -> fileService.readFile(fakePath));
+    }
+}
```
Comment (boroda4436) [inline src/test/java/com/maks/durov/amazonreviewapp/service/FileServiceTest.java]: empty line

**Review by boroda4436 — APPROVED** [review]

File: src/test/java/com/maks/durov/amazonreviewapp/service/FileServiceTest.java (hunk 2 above)
Comment (maks45) [reply src/test/java/com/maks/durov/amazonreviewapp/service/FileServiceTest.java]: done

File: src/main/resources/templates/index.html (hunk 1 above)
Comment (maks45) [reply src/main/resources/templates/index.html]: done

**Pull request merged**